### PR TITLE
API Refactor: Create MakersController

### PR DESCRIPTION
**Pull request by ConnorDY**
Also:

- commented out TEST user code previously used when doing local development
- cleaned up RequestsController

### CODE REVIEW

File: server/utils.ts [hunk 1]
```diff
@@ -5,14 +5,14 @@ import { IUser } from './interfaces';
...
-    };
-  }
+  // if (process.env.NODE_ENV === 'development') {
+  //   user = {
+  //     _id: 'TEST',
+  //     firstName: 'Test',
+  //     lastName: 'Test',
+  //     makerId: '5e781b3ee7179a17e21a89e1'
+  //   };
+  // }
```
Comment (mjoyce91) [inline server/utils.ts:15]: Is there any reason we can't keep this for development?

File: server/app.ts [hunk 2]
```diff
@@ -14,27 +14,28 @@ import { createProxyMiddleware } from 'http-proxy-middleware';
...
 };
-const csrfProtection = csurf({
-  cookie: {
-    key: 'XSRF-TOKEN',
-    path: '/',
-    httpOnly: true,
-    secure: process.env.NODE_ENV === 'production',
-    maxAge: 3600 // 1-hour
-  }
-});
```
Comment (mjoyce91) [inline server/app.ts:35]: shouldn't we keep this for production?

Comment (ConnorDY) [reply server/app.ts:35]: It was never being used.

File: server/utils.ts (hunk 1 above)
Comment (ConnorDY) [reply server/utils.ts:15]: Someone can uncomment it to re-enable it. I want to encourage people to test locally using an actual account.

Comment (mjoyce91) [reply server/utils.ts:15]: Gotcha, sounds good

**Pull request merged**